sym/README.md

```diff
@@ -0,0 +1,43 @@
+# Sym Log Destination
+
+## Overview
+
+[Sym][1] is a platform that allows you to define simple automations that turn just-in-time access policies into easy-to-operate workflows, executed in Slack. Define access flows in Terraform, customize and integrate with other systems in code, and use our API or Slack App to request and approve/deny access.
+
+This integration enables customers to send Sym audit logs directly to Datadog using a Sym Log Destination. 
+
+These logs are sent in real time for every event processed by the Sym platform, such as `request` or `approve`.
+
+## Setup
+
+### Installation
+
+To set up the Sym integration:
+1. From the Sym Datadog Integration tile, click on "Connect Accounts".
+2. Datadog will redirect you to Sym to begin the OAuth authorization flow. Enter your Sym Org ID here to continue to log in to Sym.
+3. After successfully authorizing, a `sym_log_destination` Terraform resource will display. Copy and paste this into your Sym Terraform Configuration.
+
+### Configuration
+
+For more information about configuring your Datadog Log Destination in Terraform, see the [Sym documentation][3].
+
+### Validation
+
+After you have Terraformed your Datadog Log Destination, you can confirm its existence with the following `symflow` CLI command:
+```
+ symflow resources list sym_log_destination
+```
+
+### Uninstallation
+
+- Uninstall the integration by disabling the Sym application in the OAuth Apps section of your Datadog Organization settings.
```

Comment:
The recommended flow to uninstall is for users to click the Uninstall button on the integration tile